README.md

```diff
@@ -47,10 +47,29 @@ This Ubuntu-based Docker image runs a CUPS instance that is meant as an AirPrint
 
 ### Example run env command:
 ```
-docker run --name cups --restart unless-stopped  --net host\
+docker run --name airprint --restart unless-stopped  --net host\
   -v <your services dir>:/services \
   -v <your config dir>:/config \
   -e CUPSADMIN="<username>" \
   -e CUPSPASSWORD="<password>" \
-  znetwork/cups-avahi-airprint:latest
-```
\ No newline at end of file
+  znetwork/synology-airprint:latest
+```
+
+### Example with docker componse:
+```
+version: '3.3'
```

Comment:
Meanwhile the version in docker-compose files is obsolete, so it can/should be removed.